CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Tue Mar 13 2018 Trevor Vaughan <tvaughan@onyxpoint.com> - 5.1.1-0
+- Ensure that only IPv4 is used if IPv6 is disabled on the system
```

Comment:
changelog is out of time order and doesn't  properly reflect what has been released